server/constants/prisonConfiguration.test.ts

```diff
@@ -0,0 +1,21 @@
+import getPrisonConfiguration from './prisonConfiguration'
+
+describe('Prison configuration', () => {
+  it('should return correct prison configuration for requested prisonId', () => {
+    const prisonConfiguration = getPrisonConfiguration('HEI')
+
+    expect(prisonConfiguration.prisonPhoneNumber).toBe('0300 060 6503')
```

Comment:
Do you think we should have a results file for these prison specific tests? Just thinking if a phone number changes, it'd be nice to change in one place that every test it's used?

Reply:
Not sure ...🤔 Thought about that when I was doing it, but I figured that for testing the config lookup for a `prisonId` it was enough here to do a known & unknown one. Could do this though when we add more prisons, I'll have a think!

(Scattered around the other tests is coverage for Hewell, Bristol and 'unknown' prison config but, again, wasn't planning to expand this to more prisons as it seemed like enough coverage.)